src/networking.c

```diff
@@ -3997,10 +3997,15 @@ void processEventsWhileBlocked(void) {
  * ========================================================================== */
 
 #define IO_THREADS_MAX_NUM 128
+#define CACHE_LINE_SIZE 64
+
+typedef struct __attribute__((aligned(CACHE_LINE_SIZE))) threads_pending {
+    unsigned long value;
+} threads_pending;
 
 pthread_t io_threads[IO_THREADS_MAX_NUM];
 pthread_mutex_t io_threads_mutex[IO_THREADS_MAX_NUM];
-redisAtomic unsigned long io_threads_pending[IO_THREADS_MAX_NUM];
+redisAtomic threads_pending io_threads_pending[IO_THREADS_MAX_NUM];
```

Comment:
i think `redisAtomic` should be moved into the struct?
i.e. `_atomic` of C11 be set on the struct member, rather than the array of structs.

Reply:
thanks for the suggestion, done